drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -1206,6 +1218,12 @@ static const struct i2c_driver_api i2c_it8xxx2_driver_api = {
 	.recover_bus = i2c_it8xxx2_recover_bus,
 };
 
+BUILD_ASSERT(((DT_INST_PROP(SMB_CHANNEL_B, fifo_enable) == true) &&
+	     (DT_INST_PROP(SMB_CHANNEL_C, fifo_enable) == false)) ||
+	     ((DT_INST_PROP(SMB_CHANNEL_B, fifo_enable) == false) &&
+	     (DT_INST_PROP(SMB_CHANNEL_C, fifo_enable) == true)),
+	     "FIFO2 only supports one channel of B or C.");
```

Comment:
Non-blocking comment - this will cause a build error if the user adds `/delete-property/ fifo-enable;` to both channel B and channel C.    This seems unlikely as I would expect users to just set `CONFIG_I2C_IT8XXX2_FIFO_MODE=n` if they want to disable FIFO mode.

Reply:
Blocking BUILD_ASSERT with CONFIG_I2C_IT8XXX2_FIFO_MODE can prevent users to disable FIFO mode and add "/delete-property/ fifo-enable;" to both channel B and channel C causing build error.